src/AR/objects/Instruments/Microphone.ts

```diff
@@ -67,14 +68,22 @@ export default class Microphone extends Object3D {
         .multiplyScalar(1 / 2)
         .add(this.keypoints[2].position);
 
+      // Set base of the object to the anchor point
       this.obj.position.x = anchor.x;
-      this.obj.position.y = anchor.y;
+      this.obj.position.y = anchor.y - 20;
```

Comment:
shouldn't we find a mathematic logic relative to scene instead of a magic value ?

Reply:
Yes, it will be fixed either in #54 or in #42